packages/vuetify/src/components/VNumberInput/VNumberInput.tsx

```diff
@@ -294,17 +294,15 @@ export const VNumberInput = genericComponent<VNumberInputSlots>()({
     }
 
     function onFocus () {
-      focus()
       trimDecimalZeros()
     }
 
     function onBlur () {
-      blur()
       clampModel()
     }
 
     useRender(() => {
-      const { modelValue: _, ...textFieldProps } = VTextField.filterProps(props)
+      const { modelValue: _, ...textFieldProps } = omit(VTextField.filterProps(props), ['focused'])
```

Comment:
@ikushum Is there a reason we don't need to do this on other components that call VTextField.filterProps?

![image](https://github.com/user-attachments/assets/9c6a5b63-7ebd-49df-8cab-c2a6d747b7e2)

Reply:
@johnleider 

This is because in VNumberInput `textFieldProps` was being spread into `<VTextField>` in the end but in other components, it was being spread at the beginning.

Since in `VNumberInput` it was being spread in the end, `v-model:focused={ isFocused.value }` would always be overwritten by `focused` of `textFieldProps`.

Reply:
After careful observation, I think this spread behaviour in `VNumberInput` is problematic so in the latest commit I'm now spreading it at the end and removed the omit.